### ISSUE TITLE
Does the shortcut convolutional layer has BN?

### ISSUE THREAD

**Issue by Detectionsmallvehicle** [dalgu90/resnet-18-tensorflow#9]
As is shown in common resnet-N models，if output channels > input channels，the shortcut should be a layer with Batch Normalization. However, as is shown in the programs provided, the Batch Normalization did not exist. Isn't is provided in the pre-trained pytorch model?
Thank you very much!

**Comment by jfpalngipang** [comment]
I am also wondering about the same thing. This lacks the batch norm in `_residual_block_first()`

**Comment by FangGet** [comment]
yes, batch_norm is missing for _residual_block_first.

**Comment by FangGet** [comment]
nope, the original t7 file has no batch_norm for _residual_block_first
![sss](https://user-images.githubusercontent.com/11433213/66694409-6d7bb300-ece5-11e9-9d46-d3c7eecc8308.png)
